src/mixins/amalgamation-mixin.ts

```diff
@@ -58,6 +64,14 @@ export default class AmalgamationMixin extends Vue {
     return null
   }
 
+  /** Disallow frozen business. */
+  notFrozen (business: AmalgamatingBusinessIF): AmlStatuses {
+    if (business.type === AmlTypes.LEAR && business.isFrozen) {
+      return AmlStatuses.ERROR_FROZEN
+    }
+    return null
+  }
```

Comment:
Please update the unit tests for this mixin, too.

Reply:
Should already be there Sev!